src/main/java/io/cdap/plugin/aws/s3/connector/S3Connector.java

```diff
@@ -63,7 +64,7 @@
 @Plugin(type = Connector.PLUGIN_TYPE)
 @Name(S3Connector.NAME)
 @Category("Amazon Web Services")
-@Description("Connector to browse and sample from Amazon S3.")
+@Description("Connection to browse and sample data from Amazon S3.")
```

Comment:
Should keep Connector here

Reply:
this is per request of our document writer : https://cdap.atlassian.net/browse/CDAP-18109